Campaign/public/showtime.php

```diff
@@ -290,6 +290,8 @@ public function get()
     'scheme' => 'tcp',
     'host'   => getenv('REDIS_HOST'),
     'port'   => getenv('REDIS_PORT') ?: 6379,
+],[
+    'parameters' => ['password' => env('REDIS_PASSWORD', null)],
```

Comment:
If it's possible, let's use password within the first `Predis` argument - tested on Redis 3.2 in Docker.

```
$redis = new \Predis\Client([
    'scheme' => 'tcp',
    'host'   => getenv('REDIS_HOST'),
    'port'   => getenv('REDIS_PORT') ?: 6379,
    'password' => getenv('REDIS_PASSWORD') ?: null,
]);
```

Also `env` is Laravel's function which we didn't wanted to use here. Please use PHP's `getenv` instead.

Reply:
It works this way, changed.